parser/src/powdr.lalrpop

```diff
@@ -80,39 +80,39 @@ pub PilStatement = {
 };
 
 Include: PilStatement<T> = {
-    <start:@L> "include" <file:StringLiteral> => PilStatement::Include(<>)
+    <start:@L> "include" <file:StringLiteral> => PilStatement::Include(ctx.source_ref(start), file)
 };
 
 Namespace: PilStatement<T> = {
-    <start:@L> "namespace" <name:SymbolPath> "(" <pol_degree:Expression> ")" => PilStatement::Namespace(<>)
+    <start:@L> "namespace" <name:SymbolPath> "(" <pol_degree:Expression> ")" => PilStatement::Namespace(ctx.source_ref(start), name, pol_degree)
```

Comment:
I'm wondering if we could use a macro to make these less ugly...

Reply:
On the other hand, the macro would probably also look ugly ;)

Reply:
Something like `pil_statement!(Namespace, <>)`?
I think its not too bad as is, since the ugliness is contained to the lalrpop file and makes the need for the `ParserContext` more obvious